Flaskupdate.py

```diff
@@ -0,0 +1,31 @@
+from gingerit.gingerit import GingerIt   #Initiates the Ginger IT

```

Comment:
Great Gokal, this is the flask file to connect the front and the back end. I can also see that the code has comments to explain how it functions. This is cool and will be merged to the dev branch.